src/components/Terms.vue

```diff
@@ -6,38 +6,118 @@
+        <p>  1.2. It's your own responsibility to read the Terms of Service carefully before you start to use V SYSTEMS.</p>
+        <br>
       </div>
       <div class="smalltitle">
-        <h3 align="left"> Owner and Data Controller</h3>
+        <h3 align="left">2. Definitions and Services offered
+        </h3>
       </div>
       <div class="txt"
            align="justify">
-        <p> The Terms of Use includes the terms and conditions (the “Terms” ) set forth herein which applies to the website located at www.v.systems (the “Website”, ”we”, “our" or “us” ). We remind you, before accessing to and using of the Website, you shall carefullyread and fully understand the Terms, including the Prohibited Actions, Representations and Warranties, Indemnification and Disclaimer clauses.</p>
-        <p>  If you choose to access and use the Website and any of its features (collectively, the “Services" ), you agree to abide by all the Terms set forth herein between you and us, and all the applicable laws and regulations. If you do not accept all the Terms, do not use the Website. Your access and use of the Website constitutes an implied consent of yours to the Terms.</p>
-        <p> We reserve the right, at its sole and absolute discretion, to change, modify, supplement, update, add or remove (collectively, the Changes" ) portions of the Terms, at any time, without your prior consent. The Changes are effective immediately upon posting on the Website. We will not be responsible to notify you any Changes. lt is your responsibility to check the Terms every time you access and use the Website. If you continue to use the Website following the posting of changes, it shall mean you accept and agree to the changes.</p>
+        <p>V SYSTEMS Wallet is an innovative cryptocurrency infrastructure, which consists of:</p>
+        <p>2.1. V SYSTEMS Web Wallet Application</p>
+        <p>V SYSTEMS has created means for users to access and transfer ownership of VSYS coins (such as bitcoin) under the name "V SYSTEMS Wallet...". By entering into this agreement You will receive a license to use any of the V SYSTEMS Web Wallet Application. You should only use the V SYSTEMS Web Wallet Application if you are familiar with virtual currencies. We highly recommend learning the basics of the specific virtual currency that You plan on using the V SYSTEMS Web Wallet Application before running the application.</p>
+        <p>2.1.1. You Own Your Private Keys</p>
+        <p>V SYSTEMS does not store user’s private keys, backup phrases or passwords ("Private Information"). It is very important that You backup Private Information. If You lose Your Private Information then it will not be possible for V SYSTEMS to recover it for You and You may lose access to Your VSYS coins. If You do not understand the above paragraph then we strictly recommend that You to learn more about virtual currencies before using the V SYSTEMS Web Wallet Application.</p>
```

Comment:
For "V SYSTEMS does not store user’s private keys, backup phrases or passwords ...", remove "backup" here.